src/v/security/acl.h

```diff
@@ -33,7 +33,7 @@ struct dependent_false : std::false_type {};
 // cluster is a resource type and the acl data model requires that resources
 // have names, so this is a fixed name for that resource.
 using acl_cluster_name = named_type<ss::sstring, struct acl_cluster_name_type>;
-inline const acl_cluster_name default_cluster_name("redpanda-cluster");
+inline const acl_cluster_name default_cluster_name("kafka-cluster");
```

Comment:
can we put the comment of this commit above this var too.

Reply:
fixed